content/applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst

```diff
@@ -72,213 +114,233 @@ Once the :guilabel:`Lot/Serial Number` is generated, click the blank field next
 :guilabel:`Product` to reveal a drop-down menu. From this menu, select the product to which this new
 number will be assigned.
 
-This form also provides the option to adjust the :guilabel:`Quantity`, to assign a unique
-:guilabel:`Internal Reference` number (for traceability purposes), and to assign this specific
-lot/serial number configuration to a specific website in the :guilabel:`Website` field (if working
-in a multi-website environment).
-
-A detailed description of this specific lot/serial number can also be added in the
+This form also provides the option to adjust the :guilabel:`On Hand Quantity`, to assign a unique
+:guilabel:`Internal Reference` number (for additional traceability), and to assign this specific
+lot/serial number configuration to a specific company in the :guilabel:`Company` field. A detailed
+description of this specific lot/serial number can also be added in the
 :guilabel:`Description` tab below.
 
 When all desired configurations are complete, click the :guilabel:`Save` button to save all changes.
 
-.. image:: serial_numbers/serial-numbers-new-serial-number.png
+.. image:: serial_numbers/new-serial-number.png
    :align: center
    :alt: New serial number created for existing product stock.
 
 After a new serial number has been created, assigned to the desired product, and saved, navigate
-back to the product form, by going to :menuselection:`Products --> Products`, and selecting the
-product that this newly-created serial number was just assigned to.
+back to the product form, by going to :menuselection:`Inventory app --> Products --> Products`, and
+selecting the product that this newly-created serial number was just assigned to.
 
 On that product's detail form, click the :guilabel:`Lot/Serial Numbers` smart button to view the new
 serial number.
 
-Manage serial numbers for shipping and receiving
-================================================
+.. warning::
+   If a product doesn't have a serial number assigned to it, a user error pop-up window may appear.
+   The error message states that the product(s) in stock have no lot/serial number. However, a
+   lot/serial number can be assigned to the product by making an :doc:`inventory adjustment
```

Comment:
This is odd to me— we just explained how to assign lot/serial to specific product and yet, this warning implies two things: a) that what we just did in the instructions didn't work and b) to perform a different set of steps as a "hack" to assign a lot/serial number.

Is there a way to rewrite this so it inspires more confidence in the directions or maybe is presented as a **Tip**? Could possibly remove it entirely, too. 

Do you see what I mean?